workshops/util.py

```diff
@@ -326,6 +324,9 @@ def normalize_name(name):
     for (accented, flat) in [(' ', '-')]:
         name = name.replace(accented, flat)
 
+    # remove all non-ASCII, non-hyphen chars
```

Comment:
the comment is incorrect: `.` and `'` are ASCII characters, but this removes them.